src/models/contracts/connection.ts

```diff
@@ -254,3 +254,26 @@ export class ListDatabasesResult {
 }
 
 // ------------------------------- </ List Databases Request > --------------------------------------
+
+// ------------------------------- < Connection String Request > ---------------------------------------
+
+// Get Connection String request callback declaration
+export namespace GetConnectionStringRequest {
+    export const type = new RequestType<GetConnectionStringParams, GetConnectionStringResult, void, void>('connection/getconnectionstring');
+}
+
+// Get Connection String request format
+export class GetConnectionStringParams {
+    // Connection key to lookup connection string for the provided connection Uri
+    public ownerUri: string;
+
+    // Indicates whether to include the password in the connection string
```

Comment:
If you're adding comments like this they should be in js doc format (/** ... */)

Reply:
Sure thing will update this request section. It seems other requests in this file that I was following are not like that.